src/app/page.js

```diff
@@ -21,10 +21,23 @@ const sortOptions = [
 
 export default function Page() {
   const params = useSearchParams();
+  const router = useRouter();
   const alert = params.get("alert");
   const message = params.get("message");
   const [keyword, setKeyword] = useState("");
   const [sort, setSort] = useState(sortOptions[0].value);
+  const [showAlert, setShowAlert] = useState(false);
```

Comment:
With the new changes of the redirect not having the params, this is no longer required

```suggestion
```